apps/endatix-hub/app/forms/[formId]/update-form-name.action.ts

```diff
@@ -0,0 +1,13 @@
+"use server";
+
+import { updateForm  } from "@/services/api";
+
+export async function updateFormNameAction(formId: string, formName: string) {
```

Comment:
Again just a remark. We should clean up control flow and get rid if this try/catch with the #124 . Also add typed object for each state mutation command, so we can clean up everything